### ISSUE TITLE
Runtypes don't refer to other generated runtypes

### ISSUE THREAD

**Issue by rkok** [johngeorgewright/runtyping#60]
Given the following input:

```ts
// input.ts
type HorseType = { foo: string }
type FooType = string | HorseType;
```

```yml
# runtyping.yml
- targetFile: output.ts
  sourceTypes:
    - file: input.ts
      type: HorseType
    - file: input.ts
      type: FooType
```

I would expect the output to be:

```ts
export const HorseType = Record({ foo: String, });
// ...
export const FooType = String.Or(HorseType);
```

But instead it's:

```ts
export const HorseType = Record({ foo: String, });
// ...
export const FooType = String.Or(Record({ foo: String, }));
```

Is there any way to achieve the expected output?

**Comment by johngeorgewright** [comment]
Fixed in #61

**Comment by rkok** [comment]
Thank you!
